src/Sylius/Bundle/PromotionBundle/Form/Type/CatalogPromotionAction/FixedDiscountActionConfigurationType.php

```diff
@@ -26,28 +26,19 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         $builder
             ->add('amount', MoneyType::class, [
                 'label' => 'sylius.ui.amount',
+                'constraints' => [
+                    new Type(['type' => 'numeric', 'groups' => ['sylius']]),
+                ],
```

Comment:
I'm wondering maybe the `groups` option should be passed through the constructor. Now it cannot be easily changed, as you need to remove, and add a given field.